ro-base.owl

```diff
@@ -14,16 +14,15 @@
      xmlns:swrl="http://www.w3.org/2003/11/swrl#"
      xmlns:swrla="http://swrl.stanford.edu/ontologies/3.3/swrla.owl#"
      xmlns:swrlb="http://www.w3.org/2003/11/swrlb#"
-     xmlns:terms="http://purl.org/dc/terms/"
```

Comment:
We lost DC terms.

Reply:
I just looked at ro-edit and saw that the right namespace is being used. But it's getting mysteriously changed in the build somewhere, since the IRIs are changing throughout various files.

Reply:
It can come from `core.owl` file. I'll change it.

Reply:
I changed the namespace to terms, and it worked as expected. I didn't change the annotation in the `core.owl` file.

Reply:
It looks like it fixed the issue, thanks! It's a little worrisome though; some part of the pipeline seems to be using the prefix name instead of the actual defined expansion.

Reply:
@anitacaron which version of ODK where you using for this built? dev or latest?

Reply:
I always use the latest. I don't change to dev.

Reply:
Ok, I cant see anything in the Makefile that would be able to explain this. A typical scenario that could cause something like this is cycling through jena (query --update) or using --add-prefixes in ROBOT. But I cant see anything to that end!